Resources/Prototypes/Entities/Mobs/Player/human.yml

```diff
@@ -16,6 +16,8 @@
       hair: HairBald
     - type: Loadout
       prototypes: [SyndicateOperativeGearExtremelyBasic]
+      roleLoadout: [ RoleSurvivalSyndicate ]
+
     - type: RandomMetadata
```

Comment:
formatting

Reply:
The sad part is that I did spot this, I fixed it, then I undid the fix and never fixed it again :/
Thanks